src/xhydro/extreme_value_analysis/parameterestimation.py

```diff
@@ -554,25 +600,15 @@ def return_level(
     cint_upper_data = result_upper.rename(
         {var: f"{var}_upper" for var in result_upper.data_vars}
     )
+
     data = xr.merge([result_return, cint_lower_data, cint_upper_data])
 
-    # Add coordinates for the distribution parameters and transpose to original shape (with dim -> dparams)
-    dims = [d if d != dim else "return_level" for d in ds.dims]
-    out = data.assign_coords(return_level=return_level_dim)
-    out = out.transpose(*dims)
-    out.attrs = prefix_attrs(
-        ds.attrs, ["standard_name", "long_name", "units", "description"], "original_"
-    )
-    dist = get_dist(dist)
-    attrs = dict(
-        long_name=f"Return level estimation",
-        dist=dist.name,
-        method=METHOD_NAMES[method].capitalize(),
-        return_period=return_period,
-        confidence_level=confidence_level,
-    )
-    out.attrs.update(attrs)
-    return out
+    if stationary:
+        data = data.assign_coords({"return_period": [return_period]})
```

Comment:
It can be added regardless.
```suggestion
    data = data.assign_coords({"return_period": [return_period]})
```